native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -14,21 +14,19 @@
 
 using FSTMATCH = fst::SortedMatcher<fst::StdVectorFst>;
 
-std::vector<std::pair<double, Output>> ctc_beam_search_decoder(
-    const std::vector<std::vector<double>> &probs_seq,
+std::vector<Output> ctc_beam_search_decoder(
```

Comment:
Because it complicates the bindings and adds a ton of unnecessary data copying. If we had slices, and if SWIG didn't choke on nested STL structures, it'd be much easier to keep it the way it was.